sec_certs/certificate.py

```diff
@@ -739,9 +751,14 @@ def remove_algorithms(self):
     @staticmethod
     def get_compare(vendor: str):
         vendor_split = vendor.replace(',', '') \
-            .replace('-', ' ').replace('+', ' ').replace('®', '').split()
+            .replace('-', ' ').replace('+', ' ').replace('®', '').replace('(R)', '').split()
         return vendor_split[0] if len(vendor_split) > 0 else vendor
 
+    def clean_empty_entries(self):
+        self.web_scan.algorithms = [x for x in self.web_scan.algorithms if x != {"Certificate": []}]
```

Comment:
Could the empty entries not be created at all? E.g. change the creation algorithm so that it does not pollute the algorithm list with these entries? This also created a bunch of dictionaries only to compare against them.